geometry_msgs/Pose cur_pos
geometry_msgs/Point[] obs_pos
float64 cur_speed
float64 target_speed
waypoint[] waypointAry

